README.md

```diff
@@ -0,0 +1,254 @@
+# Cypress Visual Regression with Resemble.JS
+
+[![npm](https://img.shields.io/npm/v/cypress-visual-regression-resemble-js)](https://www.npmjs.com/package/cypress-visual-regression-resemble-js)
+
+[![github actions](https://github.com/Andremoniy/cypress-visual-regression-resemble-js/workflows/Continuous%20Integration/badge.svg)](https://github.com/Andremoniy/cypress-visual-regression-resemble-js/actions)
```

Comment:
This image does not display itself ?

Reply:
Let's say: it is not ready yet. I will remove for now and investigate more.